jmh-benchmarks/src/main/java/org/apache/kafka/jmh/acl/AclAuthorizerBenchmark.java

```diff
@@ -116,4 +137,14 @@ public void tearDown() {
     public void testAclsIterator() {
         aclAuthorizer.acls(AclBindingFilter.ANY);
     }
+
+    @Benchmark
+    public void testAuthorizer() throws Exception {
+        ResourcePattern resource = new ResourcePattern(ResourceType.TOPIC, "resource-1", PatternType.LITERAL);
```

Comment:
looks like we don't have any matching prefixed ACLs for this resource. It is good to add few matching  prefixed ACLs

Reply:
good catch, thanks, let me add